src/module.ts

```diff
@@ -109,13 +109,13 @@ export namespace HeapTypeRef {
   export const I31: HeapTypeRef = 4 /* _BinaryenHeapTypeI31 */;
   export const Struct: HeapTypeRef = 5 /* _BinaryenHeapTypeStruct */;
   export const Array: HeapTypeRef = 6 /* _BinaryenHeapTypeArray */;
-  export const String: HeapTypeRef = 7 /* _BinaryenHeapTypeString */;
-  export const StringviewWTF8: HeapTypeRef = 8 /* _BinaryenHeapTypeStringviewWTF8 */;
-  export const StringviewWTF16: HeapTypeRef = 9 /* _BinaryenHeapTypeStringviewWTF16 */;
-  export const StringviewIter: HeapTypeRef = 10 /* _BinaryenHeapTypeStringviewIter */;
-  export const None: HeapTypeRef = 11 /* _BinaryenHeapTypeNone */;
-  export const Noextern: HeapTypeRef = 12 /* _BinaryenHeapTypeNoext */;
-  export const Nofunc: HeapTypeRef = 13 /* _BinaryenHeapTypeNofunc */;
+  export const String: HeapTypeRef = 8 /* _BinaryenHeapTypeString */;
```

Comment:
Let's add the missing ones :)
```suggestion
  export const String: HeapTypeRef = 7 /* _BinaryenHeapTypeExn */;
  export const String: HeapTypeRef = 8 /* _BinaryenHeapTypeString */;
```
Plus: Add `_BinaryenHeapTypeExn` to `src/glue/binaryen.(js|dts)`

Reply:
Yeah, I noticed the missing `Exn`, but I wasn't quite sure what to do. I'll get on it.